packaging/obs/prometheus-sap_host_exporter.spec

```
@@ -58,23 +58,26 @@ go build -mod=vendor \
 install -D -m 0755 %{shortname} "%{buildroot}%{_bindir}/%{shortname}"
 
 # Install the systemd unit
-install -D -m 0644 %{shortname}.service %{buildroot}%{_unitdir}/%{name}.service
+install -D -m 0644 %{shortname}@.service %{buildroot}%{_unitdir}/%{name}@.service
+
+# Install the default config
+install -D -m 0600 default.yaml "%{buildroot}%{_bindir}/etc/%{shortname}/default.yaml"
```

I don't think this would work, the source file is in the `doc` directory. Also, per the above comment, this should probably be:
```suggestion
install -D -m 0600 doc/%{shortname}.yaml "%{buildroot}%{_bindir}/etc/%{shortname}/default.yaml"
```